auction9-frontend/src/app/data-table/data-table.component.ts

```diff
@@ -1,10 +1,11 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, SimpleChanges } from '@angular/core';
```

Comment:
Is `SimpleChanges` neccessary, I can not see that is in use?
Also I saw that `Input` ,`ChangeDetectorRef`,`AfterViewInit`, `ViewChild` are from the same module @angular/core, so it can be merge into one import statements, for example into this one from line 1.

Reply:
SimpleChanges is not necessary and removed now, I used it for some testing but switched to use 'ChangeDetectorRef' instead. Also all modules from '@angular/core' merged into one import statement.